### DISCUSSION TITLE
Docker Hub ratelimiting prevents updates from private Docker registries

### DISCUSSION THREAD

**Discussion by jokay** [renovatebot/renovate#11109] [Request Help] [answered]
Given a repository with a lot of `docker-compose.yml` which uses images from `hub.docker.com` as well as images from private Docker registries.

If the ratelimiting error for `hub.docker.com` occurs *(Response code 429 (Too Many Requests))*, the images from private Docker registries not having such a ratelimiting won't be updated as well.

I would expect the ratelimiting not affecting updates from private Docker registries?

**Comment by viceice** [comment, marked as answer]
See https://docs.renovatebot.com/configuration-options/#abortIgnoreStatusCodes

**Reply by rarkins** [reply]
I suggest you don't want to continue after a failure like that because you may end up with existing PRs autoclosed or dependencies removed from existing groups

**Reply by jokay** [reply]
Good point.
